test/teyu_test.rb

```diff
@@ -1,13 +1,11 @@
-require "test_helper"
+require_relative "test_helper"
 
 class TeyuTest < Test::Unit::TestCase
   def test_that_it_has_a_version_number
     refute_nil ::Teyu::VERSION
   end
 
-  # FYI: name of args type are referenced Method#parameters docs.
-  # https://docs.ruby-lang.org/ja/latest/method/Method/i/parameters.html
```

Comment:
I don't think we should follow such non-intuitive names.
I guess it uses such non-intuitive names just because ruby can not break compatibilities.